packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -563,6 +564,21 @@ export class QuoteRequestor {
                 this._warningLogger(order, 'Expiry too soon in RFQ-T firm quote, filtering out');
                 return false;
             } else {
+                const fillRatio = order.takerAmount.div(assetFillAmount);
```

Comment:
why not `order.takerAmount < assetFillAmount`?

Reply:
discussed offline - mostly to find intent when a MM wants to send 100%, but for some reason sent slightly less